src/lib.rs

```diff
@@ -3,6 +3,11 @@
 #![warn(rust_2018_idioms)]
 #![doc(html_root_url = "https://docs.rs/indexmap/1/")]
 #![no_std]
+#![cfg_attr(
+    feature = "debugger_visualizer",
+    feature(debugger_visualizer),
+    debugger_visualizer(natvis_file = "../debug_metadata/indexmap.natvis")
+)]
```

Comment:
I'm personally very wary of using any unstable features in a stable crate, even behind an optional feature.

Reply:
I understand, the goal is to push the `debugger_visualizer` feature to stabilization relatively soon by having it provide value to crates in the wild today as with any unstable feature.

As you said it is gated behind an optional feature but you as the crate maintainer would have to potentially take on an extra cost which I fully understand could be worrisome.

Reply:
Well, I gave approval for CI to run, at least, so it can be a proof of concept for the feature.

Reply:
Great! I see the CI was green and tests passed. Thanks for approving the CI job.

Reply:
@cuviper Do you have any reservations about taking in this change now that the CI shows as green?

Reply:
Passing CI is necessary, but doesn't change the maintenance aspect. If the unstable feature changes, as it is allowed, then we'll have to update again to fix our own CI. That's especially true if any users (on nightly) start depending on it (providing "value in the wild").

For ad hoc demonstrations in the meantime, you could use cargo's patch mechanisms to point at your git fork.